content/en/docs/setup/install/multicluster/shared/index.md

```diff
@@ -71,7 +72,11 @@ Decide how to expose the main cluster's Istiod discovery service to
 the remote clusters. Choose between **one** of the two options below:
 
 * Option (1) - Use the `istio-ingressgateway` gateway shared with data traffic.
-* Option (2) - Use a cloud provider’s internal load balancer on the Istiod service.
+
+* Option (2) - Use a cloud provider’s internal load balancer on the Istiod service. See
+[Kubernetes Internal Load Balancer docs](https://kubernetes.io/docs/concepts/services-networking/service/#internal-load-balancer)
+and your cloud provider's documentation for additional requirements and restrictions that may apply when using an internal load balancer
+between clusters.
```

Comment:
```suggestion
* Option (2) - Use a cloud provider’s internal load balancer on the Istiod
  service. For additional requirements and restrictions that may apply when using
  an internal load balancer between clusters, see
  [Kubernetes internal load balancer documentation](https://kubernetes.io/docs/concepts/services-networking/service/#internal-load-balancer)
  and your cloud provider's documentation.
```

Description of my recommended changes:
* Reverse the order of the clauses to put the condition first. (https://developers.google.com/style/clause-order)
* Lowercase "internal load balancer" to match the case used on the linked-to page.
* Change "docs" to "documentation". (https://developers.google.com/style/word-list#documentation)